frontend/src/components/CarCard.jsx

```diff
@@ -5,19 +5,25 @@ import { Link } from "react-router-dom";
 
 export default function CarCard() {
   const [user, setUser] = useState([]);
+  const [date, setDate] = useState("");
 
   useEffect(() => {
     axios.get("http://localhost:5000/user/1/vehicle").then(({ data }) => {
```

Comment:
You should put this in your `conf.js`, because it will break when we deploy this code in production